ksql-functional-tests/src/test/java/io/confluent/ksql/test/tools/KsqlTestingToolTest.java

```diff
@@ -80,46 +79,34 @@ public void shouldUseAndCloseTestExecutor() throws Exception {
   }
 
   @Test
-  public void shouldFailWithIncorrectArgs() throws UnsupportedEncodingException {
-
-    // Given:
-    System.setSecurityManager(new TestSecurityManager());
-
+  public void shouldFailWithIncorrectTest() throws IOException {
     // When:
-    try {
-      KsqlTestingTool.loadAndRunTests(new String[]{"foo"});
-    } catch (final Exception e) {
-      assertThat(e, instanceOf(TestSecurityManager.ExitSecurityException.class));
-      final TestSecurityManager.ExitSecurityException exitSecurityException = (TestSecurityManager.ExitSecurityException) e;
-      assertThat(exitSecurityException.getStatus(), equalTo(-1));
-      assertThat(errContent.toString("UTF-8"), equalTo("Failed to start KSQL testing tool: foo (No such file or directory)\n"));
-    }
-  }
+    KsqlTestingTool.runWithTripleFiles(
+        "src/test/resources/test-runner/incorrect-test/statements.sql",
+        "src/test/resources/test-runner/incorrect-test/input.json",
+        "src/test/resources/test-runner/incorrect-test/output.json");
 
+    // Then:
+    assertThat(errContent.toString("UTF-8"),
+        containsString("Test failed: Expected <1001, 101> with timestamp=0 but was <101, 101> with timestamp=0\n"));
+  }
 
-  static class TestSecurityManager extends SecurityManager {
-
-    final class ExitSecurityException extends SecurityException {
-      private final int status;
+  @Test
+  public void shouldFailWithIncorrectArgs() throws UnsupportedEncodingException {
```

Comment:
This test is empty.

Reply:
Oops, yes. the TestOptions ensures the args are correct. Removed this one.